contrib/debian/watch

```diff
@@ -0,0 +1,2 @@
+version=4
+https://zfsonlinux.org/ .*zfs-([\d\.]+)\.tar\.gz$ debian uupdate
```

Comment:
I don't think we need this either. It's for uscan to check for and download upstream releases, but we are upstream.

Reply:
Removed.